json-schema/python/CHANGELOG.md

```diff
@@ -22,4 +22,9 @@
 - Added cpu-usage and memory-usage header constants for performance-message notifications.
 
 # 0.3.1
-- Package dependency update
\ No newline at end of file
+- Package dependency update
+
+# 0.3.2
```

Comment:
probably 0.4.0 - you have added new fields and changed some type

Reply:
fixed